src/MapExplorer.ls

```diff
@@ -23,12 +29,58 @@ package
     {
         private var _map:Map;
         private var _flyer:MapFlyer;
-        private var _list:List;
+        private var _listAttractions:List;
+        private var _listCategories:List;
+        private var _detailsView:Shape;
         private var _data:MapData;
         private var _timer:Timer;
 
+        private var _detailsTriggerTime:Number = NaN;
+
         public var onIdle:IdleDelegate;
 
+        public function gotoCategories():void
+        {
+            _listCategories.visible = true;
+            _listCategories.selectedIndex = -1;
+            _listAttractions.visible = false;
+            _listAttractions.selectedIndex = -1;
+            _detailsView.visible = false;
+        }
+
+        public function gotoAttractions():void
+        {
+            _listCategories.visible = false;
+            _listCategories.selectedIndex = -1;
+            _listAttractions.visible = true;
+            _listAttractions.selectedIndex = -1;
+            _detailsView.visible = false;
+        }
+
+        public function gotoDetails(dict:Dictionary.<String, Object>):void
+        {
+            _listCategories.visible = false;
+            _listCategories.selectedIndex = -1;
+            _listAttractions.visible = false;
+            _listAttractions.selectedIndex = -1;
+            _detailsView.visible = true;
```

Comment:
Probably better if we just have a `resetView` func that we call and then set just the things we need?